tasks/uglify.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = function (grunt) {
+	return {
+		build: {
+			options: {
+				mangle: true,
```

Comment:
#139 Is tested and functional minification. I am requiring this for a project I am working on so the quicker it is accepted the better, hence producing this pull request now :)